bundle/manifests/certification.redhat.com_operatorpipelines.yaml

```diff
@@ -58,6 +70,10 @@ spec:
                 description: The name of the secret containing the pyxis api secret
                   expected by the pipeline
                 type: string
+            required:
+            - applyCIPipeline
+            - applyHostedPipeline
+            - applyReleasePipeline
```

Comment:
You can use the `oneOf` keyword here, and have your CRD only contains one pipeline mode. This will ensure that you don't end up with CRDs whose two modes are set.

Reply:
The user can select multiple pipelines, specifically if it for a RH internal cluster running the certification process. That cluster would need both the `hosted` and `release` pipeline. We wanted to do some sort of `depends on` but there doesn't seem a really good way to do that with descriptors for the UI. We might be able to do something more with the UI around `advanced` features, but that's just look and feel and the CR would still have these values as required. I hope this is clear, but maybe there is a better path forward that we are not seeing.